plugins/meta/tuning/tuning.go

```diff
@@ -90,6 +164,26 @@ func cmdAdd(args *skel.CmdArgs) error {
 				return err
 			}
 		}
+
+		var err error
+
+		if tuningConf.Mac != "" {
+			if err = changeMacAddr(args.IfName, tuningConf.Mac); err == nil {
+				err = updateResultsMacAddr(*tuningConf, args.IfName, tuningConf.Mac)
+			}
+		}
+
+		if tuningConf.Promisc != false {
+			if err = changePromisc(args.IfName, true); err != nil {
+				return err
+			}
+		}
+
+		if tuningConf.Mtu != -1 {
```

Comment:
Since 0 is the golang default value, and a 0 MTU is invalid, you can just use that.

Reply:
Fixed. Thanks!